migrations/codeinsights/1656608833/up.sql

```diff
@@ -0,0 +1,4 @@
+-- Perform migration here.
+--
+ALTER TABLE IF EXISTS insight_series
+    ADD COLUMN IF NOT EXISTS backfill_attempts INT;
```

Comment:
```suggestion
    ADD COLUMN IF NOT EXISTS backfill_attempts INT NOT NULL DEFAULT 0;
```

Any reason we can't have it be default here?

Reply:
I debated but for all existing records it's not really 0 it's 🤷, easier if i default it though

Reply:
The only reason I suggested attempts instead of retries is that we can't actually track retries if something fails or is terminated in the middle (ex. a rollout interrupts the backfill), assuming we stamp at the end. As long as we stamp at the start any name works.

Reply:
yeah I'm fine with it being 0 i'll default and convert the ptr